test/RibbonThetaVault.ts

```diff
@@ -109,7 +119,7 @@ describe("RibbonThetaVault", () => {
     chainlinkPricer: CHAINLINK_WETH_PRICER_NEW[chainId],
     deltaFirstOption: BigNumber.from("1000"),
     deltaSecondOption: BigNumber.from("1000"),
-    deltaStep: BigNumber.from(chainId === CHAINID.AVAX_MAINNET ? "10" : "100"),
+    deltaStep: BigNumber.from(chainId === CHAINID.ETH_MAINNET ? "100" : "10"),
```

Comment:
should we do delta step of $1 for NEAR (at $17 right now)

Reply:
AVAX trades at ~$100, so we sell calls +- $10
ETH trades at ~$4000, so we sell calls +- $100

Reply:
Oh on Avalanche, it's really AVAX as WETH address.
Aurora on the other hand, uses ETH as it's native token (not AURORA)